CHANGELOG.md

```diff
@@ -59,22 +60,33 @@ All notable changes to this project will be documented in this file.
                 * `lcr_open_short_load_compensation_data_source`
                 * `lcr_open_susceptance`
                 * `lcr_short_compensation_enabled`
+                * `lcr_short_custom_cable_compensation_enabled`
                 * `lcr_short_reactance`
                 * `lcr_short_resistance`
                 * `lcr_source_delay_mode`
                 * `lcr_stimulus_function`
                 * `lcr_voltage_amplitude`
             * Enums added:
+<<<<<<< HEAD
```

Comment:
you need to fix this @olsl21

Reply:
Thanks for spotting this, I've fixed it in my latest commit